.github/workflows/test.yml

```
@@ -6,33 +6,121 @@ on:
   pull_request:
     branches: [ main, master ]
 
+# Limit concurrent runs of the same workflow
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   test:
     runs-on: ubuntu-latest
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)